README.md

```diff
@@ -1,11 +1,21 @@
 # DCISolver - Dynamic Control of Infeasibility Solver
 
-![CI](https://github.com/JuliaSmoothOptimizers/DCISolver.jl/workflows/CI/badge.svg?branch=master)
-[![codecov](https://codecov.io/gh/JuliaSmoothOptimizers/DCISolver.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaSmoothOptimizers/DCISolver.jl)
 [![GitHub](https://img.shields.io/github/v/release/JuliaSmoothOptimizers/DCISolver.jl.svg?style=flat-square)](https://github.com/JuliaSmoothOptimizers/DCISolver.jl/releases)
-[![docs-latest](https://img.shields.io/badge/docs-latest-3f51b5.svg)](https://JuliaSmoothOptimizers.github.io/DCISolver.jl/latest)
-[![docs-dev](https://img.shields.io/badge/docs-dev-3f51b5.svg)](https://JuliaSmoothOptimizers.github.io/DCISolver.jl/dev)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.4742979.svg)](https://doi.org/10.5281/zenodo.4742979)
+
+| **Documentation** | **CI** | **Coverage** | **DOI** |
+|:-----------------:|:------:|:------------:|:-------:|
+| [![docs-stable][docs-stable-img]][docs-stable-url] [![docs-dev][docs-dev-img]][docs-dev-url] | [![build-ci][build-ci-img]][build-ci-url] | [![codecov][codecov-img]][codecov-url] | [![doi][doi-img]][doi-url] |
+
+[docs-stable-img]: https://img.shields.io/badge/docs-stable-blue.svg
+[docs-stable-url]: https://JuliaSmoothOptimizers.github.io/DCISolver.jl/stable
```

Comment:
https://JuliaSmoothOptimizers.github.io/DCISolver.jl/stable doesn't exist yet, but I guess it is because there aren't been any new release since we setup the doc.

Reply:
Yes, that's right. It should work at the next release.